json-rpc/src/util.rs

```diff
@@ -98,6 +96,75 @@ impl SdkLang {
     }
 }
 
+pub static SDK_VERSION_REGEX: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r"\b([0-3])\.(\d{1,2})\.(\d{1,2})\b").unwrap());
+pub static DEFAULT_SDK_VERSION: Lazy<SdkVersion> = Lazy::new(|| SdkVersion {
+    major: 0,
+    minor: 0,
+    patch: 0,
+});
```

Comment:
This isn't necessary, just implement the Default trait (by doing #[derive(Default)]) and use that instead.